test/test_jit.py

```diff
@@ -11577,6 +11577,27 @@ def forward(self, x):
         FileCheck().check_not("value=<Tensor>").check_count("aten::mm", 2).check("aten::add") \
             .run(str(tm.graph))
 
+    def test_op_dtype(self):
+
+        def check_equal_and_dtype(a, b):
+            self.assertEqual(a, b)
+            self.assertEqual(a.dtype, b.dtype)
+
+        def fn():
+            a = torch.arange(10)
```

Comment:
maybe throw in a bool or two here.

Reply:
JIT doesn't allow promotion of bools to scalars, added in test_torch